<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtlabors::MainWindow</class>
 <widget class="QMainWindow" name="qtlabors::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labors</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="filter_label">
        <property name="text">
         <string>Filter: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filter_edit"/>
      </item>
      <item>
       <widget class="QPushButton" name="filter_clear">
        <property name="text">
         <string>Clear</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="gridview_tabs">
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="suspend_action"/>
   <addaction name="resume_action"/>
  </widget>
  <action name="suspend_action">
   <property name="icon">
    <iconset theme="media-playback-pause">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Suspend</string>
   </property>
   <property name="toolTip">
    <string>Suspend DF so data can be accessed</string>
   </property>
  </action>
  <action name="resume_action">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="toolTip">
    <string>Resume DF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>filter_clear</sender>
   <signal>clicked()</signal>
   <receiver>filter_edit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>748</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
